apps/site/pages/docs/index.mdx

```diff
@@ -1,3 +1,33 @@
+<header className="hero">
+
 # FastStore
 
-Here is where you will have the introduction documentation page.
+The ultimate toolkit for building blazing-fast storefronts.
+
+</header>
+
+## What is FastStore
+
+[FastStore](https://github.com/vtex/faststore) is a light yet powerful **set of libraries** made for developers who want to explore their knowledge and creativity building comprehensive and unique ecommerce solutions. It supports integrations with headless ecommerce platforms, allows meaningful analytics capabilities, and gives developers back the freedom of customization.
+
+## What is included
+
+That is possible because FastStore was cleverly split into **a UI, SDK and API packages.** The UI package is a **CSS framework-agnostic** library that crafts the behavior of React components into data attributes. The SDK is an extensible state management library that handles all meaningful states an ecommerce store might have, such as:
```

Comment:
the components are missing, UI is just the layer that styles.

Reply:
hmm, should we also explain the `core` package here?

Here we need to explain:
`components` package -> components without style. (the old "ui" package)
`ui` package -> components with the structure and minimal style

Reply:
What about:

> That is possible because FastStore was cleverly split into **Components, UI, SDK and API packages.** The `Components` package is a library that crafts the behavior of React components into data attributes. The SDK is an extensible state management library that handles all meaningful states an ecommerce store might have, such as:
> 
> - **Session** - handles auth, region, locale, and currency data.
> - **Cart** - handles the addition, remotion, update, and current state of a store cart.
> - **Component** - handles the state of the components of a store (e.g., display/hide mini cart).
> - **UI** - adds strucutural styles to the components.
> - **Search** - handles the state of the faceted search (e.g., apply filters and sort).